### PR TITLE
Fix a thing or two under test

### PR DESCRIPTION
**Pull request by ArangoGutierrez**
This patch fix typo in a file name
Updates default image on e2e tests
Adds an extra file that should be ignored to gitignore

Signed-off-by: Carlos Eduardo Arango Gutierrez <carangog@redhat.com>

### CODE REVIEW

File: test/e2e/node_feature_discovery.go [hunk 1]
```diff
@@ -44,8 +44,8 @@ import (
 )
 
 var (
-	dockerRepo    = flag.String("nfd.repo", "quay.io/kubernetes_incubator/node-feature-discovery", "Docker repository to fetch image from")
-	dockerTag     = flag.String("nfd.tag", "e2e-test", "Docker tag to use")
+	dockerRepo    = flag.String("nfd.repo", "k8s.gcr.io/nfd/node-feature-discovery", "Docker repository to fetch image from")
+	dockerTag     = flag.String("nfd.tag", "v0.6.0", "Docker tag to use")
```
Comment (marquiz) [inline test/e2e/node_feature_discovery.go:48]: Hmm, we should probably use `gcr.io/k8s-staging-nfd/node-feature-discovery:master` as the default here as a past release is not guaranteed to be compatible with tests from `master`. Another option would be to leave the both arg defaults empty

Comment (ArangoGutierrez) [reply test/e2e/node_feature_discovery.go:48]: I like master as default, the `Makefile` uses a proper flag to point it to the dirty git stage (when developing something) 
I was here really thinking in the casual users trying to understand the project.
will set it to `master`

File: test/e2e/node_feature_discovery.go [hunk 2]
```diff
@@ -45,7 +45,7 @@ import (
 
 var (
 	dockerRepo    = flag.String("nfd.repo", "k8s.gcr.io/nfd/node-feature-discovery", "Docker repository to fetch image from")
-	dockerTag     = flag.String("nfd.tag", "v0.6.0", "Docker tag to use")
+	dockerTag     = flag.String("nfd.tag", "master", "Docker tag to use")
```
Comment (marquiz) [inline test/e2e/node_feature_discovery.go:48]: You need to change `nfd.repo`, too as the production repo `k8s.gcr.io/nfd/node-feature-discovery`  does not have `master` tag

File: test/e2e/node_feature_discovery.go (hunk 1 above)
Comment (marquiz) [reply test/e2e/node_feature_discovery.go]: Mm, that's a fair point. This is just yet another place which needs to be modified when creating a release branch. I prolly need to create a helper script for this 🤔

File: test/e2e/node_feature_discovery.go (hunk 2 above)
Comment (ArangoGutierrez) [reply test/e2e/node_feature_discovery.go:48]: oh staging .... sorry missed that part

**Comment by marquiz** [comment]
BTW, created script for helping with the release preparations: #397

**Comment by marquiz** [comment]
Looks good, could you squash these into one? I'd like to preserve the commit msg of the first commit 😄

**Comment by ArangoGutierrez** [comment]
> Looks good, could you squash these into one? I'd like to preserve the commit msg of the first commit smile

Done!

**Pull request merged**